### PR TITLE
Friendly fire checkbox

### PR DESCRIPTION
**Pull request by xamionex**
I tested it locally, it worked :p
Feel free to reject if you want a different implementation tho
I just added a checkbox to the hosts lobby menu
image

### CODE REVIEW

File: Story/StoryMenu.cs [hunk 1]
```diff
@@ -591,6 +595,10 @@ public void SetChecked(CheckBox box, bool c)
                 {
                     resetSave = !resetSave;
                 }
+                if (idstring == "FRIENDLYFIRE")
+                {
+                    gameMode.friendlyFire = !gameMode.friendlyFire;
```
Comment (henpemaz) [inline Story/StoryMenu.cs:600]: this really should be `gameMode.friendlyFire = c;` shouldn't it

Comment (xamionex) [reply Story/StoryMenu.cs]: worked 👍

**Comment by 6fears7** [comment]
Thanks for this. This menu is going to be deprecated next release in favor of a rework that uses the real story menu. 

I will take your commit and essentially cherry pick it off into the rework branch which is nearly done and will give you credit in the patch notes. This PR will be closed when that change is merged.

**Comment by xamionex** [comment]
> Thanks for this. This menu is going to be deprecated next release in favor of a rework that uses the real story menu.
> 
> I will take your commit and essentially cherry pick it off into the rework branch which is nearly done and will give you credit in the patch notes. This PR will be closed when that change is merged.

Yeah I saw that, thank you for reworking the menu as right now slugbase implementation is really hard with the current menu ;p

**Comment by 6fears7** [comment]
Thanks for the work on this

**Pull request closed without merging**